src/pages/LetterStoragePage/components/DeleteBottomSheet.tsx

```diff
@@ -1,60 +1,61 @@
 import { toast } from 'react-toastify';
 import { useMutation, useQueryClient } from '@tanstack/react-query';
-import { isAxiosError } from 'axios';
 import BottomSheet from '@/components/BottomSheet';
 import Button from '@/components/Button';
 import letterAPI from '@/api/letter/apis';
 import letterOptions from '@/api/letter/queryOptions';
-import ERROR_RESPONSES from '@/constants/errorMessages';
+import useBoolean from '@/hooks/useBoolean';
+import LoadingSpinner from '@/components/LoadingSpinner';
 
-interface DeleteBottomSheetProsp {
-  value: boolean;
-  on: () => void;
-  off: () => void;
+interface DeleteBottomSheetProsp extends ReturnType<typeof useBoolean> {
   letterId: number;
-  type?: 'Letter' | 'Send';
+  modalOff: () => void;
```

Comment:
p5;
함수에 대해서는 동사를 앞에 두는 네이밍으로 동작을 명시하는 것도 좋을 것 같은데 어떻게 생각하시나요!?

```suggestion
  offModal: () => void;
```